c-gull/Cargo.toml

```diff
@@ -11,19 +11,20 @@ repository = "https://github.com/sunfishcode/mustang"
 edition = "2021"
 
 [dependencies]
-rustix = { version = "0.36.1", default-features = false, features = ["fs", "itoa", "net", "param", "process", "rand", "termios", "thread", "time"] }
 # We use the libc crate for C ABI types and constants, but we don't depend on
 # the actual platform libc.
 libc = { version = "0.2.138", default-features = false }
-errno = { version = "0.2.8", default-features = false }
-c-scape = { path = "../c-scape", version = "0.6.1", features = ["std"] }
-tz-rs = "0.6.11"
-printf-compat = "0.1.1"
-log = { version = "0.4.14", default-features = false }
+c-scape = { path = "../c-scape", version = "0.6.1", default-features = false }
```

Comment:
Why does c-scape need `default-features = false`?

If that were removed, could we also remove the c-scape/default below?

Reply:
c-scape by default builds with std which propagates down to its dependencies such as rustix. I was trying to avoid breaking any of the current default features.

Reply:
Looked into it a little more, c-gull also by default builds with `resolve` which relies on `resolve-sync` which is a library that uses std.

Reply:
Should we move resolve out into c-gull then? I think that makes sense, but don't know offhand if there's some subtle dependency there. We could also do that in a separate PR.

Reply:
Ya, moving resolve it into c-gull seems like the right call. I can do that in a separate PR. I went through the git history and resolve was around before c-gull existed. I am not sure if there was a reason it wasn't moved to c-gull, but simplest guess is it just got missed.